sl/content.md

```diff
@@ -0,0 +1,89 @@
+# Scientific Linux
+
+%%LOGO%%
+
+This image containers for [Scientific Linux](http://www.scientificlinux.org/).
+
+The source for each release is in [its own branch](http://github.com/scientificlinux/sl-docker/branches) and contains the kickstart used to build the image.
+
+These images are maintained by the Scientific Linux Team in [github](https://github.com/scientificlinux/sl-docker).
+
+## Container Tags
+
+A tag will be provided for each of the [maintained major releases](http://www.scientificlinux.org/downloads/sl-versions/) of Scientific Linux (`6`, `7`).
+
+The `latest` tag will track the highest version numbered release of Scientific Linux.
+
+Updated images will be released roughly once per month.
+
+## Build Logs
+
+The SL docker containers are built on our internal build system and packaged at [Docker Cloud](https://cloud.docker.com/app/scientificlinux/repository/docker/scientificlinux/sl/builds/).
```

Comment:
This link appears to be throwing a 500 error. :confused:

Reply:
Also, maybe swap "docker containers" for "root file system for the docker images"?  Since the resulting images are actually built via the official-images build servers.